src/parser.ts

```diff
@@ -4,6 +4,11 @@ import * as ts from 'typescript';
 
 import { buildFilter } from './buildFilter';
 
+// We'll use the currentDirectoryName to trim parent fileNames
+const currentDirectoryPath = process.cwd();
+const currentDirectoryParts = currentDirectoryPath.split('/');
```

Comment:
Maybe it would make sense to use `path.sep` instead of `'/'` only (https://nodejs.org/api/path.html#path_path_sep).